src/app/api/index.ts

```diff
@@ -69,48 +73,72 @@ class Client implements API {
         const taskMap : SheriffTaskMap = arrayToMap(tasks,(t)=>t.id);
         return Promise.resolve(taskMap);
     }
+
+    async createSheriff(newSheriff: Sheriff): Promise<Sheriff> {
+        await randomDelay();
+        
+        if(!newSheriff.imageUrl){
+            let randomNumber = Math.floor(Math.random() * 86) + 11; 
+            let randomSheriff = await fetchRandomPeople(randomNumber);
```

Comment:
The only issue with this is that we're fetching potentially many records, but only selecting one of them and cherrypicking the picture.  might make sense to create a fetchRandomPerson call that can do this.  But since this is just a mock api, it's not really a huge deal either way.

Reply:
removed the call to fetchRandomPeople and added a direct call to get an image using the randomNumber